Dockerfile

```diff
@@ -20,6 +20,19 @@ RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
     uv sync --frozen --no-install-project
 
+ENV JUST_VERSION=1.40.0
+ENV JUST_INSTALLER_URL="https://raw.githubusercontent.com/casey/just/3884a4e68395b46c4b3eed694b09955f65b79fcc/www/install.sh"
+ENV JUST_INSTALLER_SHA256SUM="2f811850e7833bf2191df55683f861d09b8a9cd2d1aac5f2adff597b3d675aa4  install.sh"
+
+RUN apt install curl \
+    && curl --proto '=https' --tlsv1.2 -sSf ${JUST_INSTALLER_URL} -o install.sh \
+    && echo ${JUST_INSTALLER_SHA256SUM} > install.sha256 \
+    && sha256sum -c install.sha256 \
+    && chmod +x install.sh \
+    && ./install.sh --tag ${JUST_VERSION} --to /usr/local/bin \
+    && rm install.sh install.sha256 \
+    && apt remove -y curl
+
```

Comment:
curious: did `apt install just` not work? One of the promising advantages of Just is that it exists in many distros.

Reply:
https://packages.debian.org/search?keywords=just

There is no bookworm package. I don't like how complicated it turned out without the package but the package wasn't an option. I'm open for recommendations.

Reply:
oh no! yeah that sucks

I haven't been using just inside containers, so I hadn't noticed it wasn't part of the Python images.

Maybe getting rid of just from the container is the best solution for projects bootstrapping in 2025. Hopefully, they figure it out.

Btw I think it's fine to try out the experiment, really nice work to figure out how to get it built in Docker :clap:

Reply:
> I haven't been using just inside containers, so I hadn't noticed it wasn't part of the Python images.

> Maybe getting rid of just from the container is the best solution for projects bootstrapping in 2025.

With make, I wasn't putting it in the container. A coworker of mine recommended this workflow, "Commands should be working inside and outside of the container", which made sense to me, and here we are. I can remove that option if people think it's increasing the complexity too much but I feel like that "works everywhere" approach was a nice touch.

Reply:
That's true! If there is a need to be running commands from inside the container, for sure we don't wanna bother with alternative commands. Needs to all be the same.

Btw thinking of how you found Hugo on Pypi, I also found Just: https://pypi.org/project/just-bin/

Reply:
> Btw thinking of how you found Hugo on Pypi, I also found Just: https://pypi.org/project/just-bin/

Oh that's nice, and it makes sense.  With the latest developments on the Hugo project, I think we don't need to develop this project anymore, but that was a nice bit of info, thank you.